src/dosext/mfs/mfs.c

```diff
@@ -4041,7 +4041,10 @@ static int dos_fs_redirect(struct vm86_regs *state)
       update_seek_from_dos(sft_position(sft), &f->seek);
       cnt = WORD(state->ecx);
       if (cnt) {
```

Comment:
Will something simpler work here, like, say,
`if (cnt && sft_position(sft) != -1)`
?

Reply:
Probably would but I'd rather keep it as is.